devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -339,6 +376,7 @@ private encap(cmd, endpoint = null) {
 }
 
 private addChildSwitches(numberOfSwitches) {
```

Comment:
I think you should utilize `Child Switch` DTH instead of `Child Metering Switch` as a child devices handler for your device.

Reply:
When you do, remember about including `childOn` and `childOff` methods here, which, for now, are missing.

Reply:
Changed to Child Switch and added methods as recommended.